apis/config/v1beta3/types.go

```diff
@@ -120,6 +120,21 @@ type LoadVariationRiskBalancingArgs struct {
 	SafeVarianceSensitivity *float64 `json:"safeVarianceSensitivity,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:defaulter-gen=true
+
+// LowRiskOverCommitmentArgs holds arguments used to configure LowRiskOverCommitment plugin.
+type LowRiskOverCommitmentArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Common parameters for trimaran plugins
+	TrimaranSpec `json:",inline"`
+	// Smoothing window size for usage data in the load watcher
+	SmoothingWindowSize *int64 `json:"smoothingWindowSize,omitempty"`
+	// Weight of risk due to limit
+	RiskLimitWeight *float64 `json:"riskLimitWeight,omitempty"`
```

Comment:
so IIUC this weight is to configure the ratio of riskLimits, and (1 - RiskLimitWeight) is the ratio of riskLoad? If so, I'd suggest to make their semantics explicit like:

```yaml
- strategies:
  - foo
    - foo-specific-arg: 
    - weight: 0.5
  - bar
    - bar-specific-arg: 
    - weight: 0.5
```

Reply:
Typically, cpu can be overcommitted more tolerantly than memory/storage, so can we add args to distinguish the importance/weight of individual resources?

Reply:
Thanks, done.